lib/task-data/schemas/install-centos.json

```diff
@@ -0,0 +1,19 @@
+{
+    "$schema": "rackhd/schemas/v1/task-schema",
+    "id": "rackhd/schemas/v1/tasks/install-centos",
+    "copyright": "Copyright 2016, EMC, Inc.",
+    "title": "Install CentOS",
+    "description": "The parameters for CentOS installation",
+    "describeJob": "Job.Os.Install",
+    "allOf": [
```

Comment:
@brianparry : I see the redfish schema (https://github.com/RackHD/on-http/blob/master/static/DSP8010_1.0.0/json-schema/redfish-schema.1.0.0.json#L7) does in this way, so I copy it because I think it is more simple.
Meanwhile, I checked both syntax works well for the validator.